IAMOTZ/ievents
# #153221814 Set up React

#### What does this PR do?
It set up a react development environment to develop the front end of the application
#### Description of Task to be completed?
Set up react-redux development environment
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- Run `npm run react:dev`
- visit the port where the webpack-dev-server started and the component rendered in index.js would be displayed there
- Modify the content in `indes.js` and the change would appear in the browser as soon as you save the file.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#153221814
#### Screenshots (if appropriate)
![screencapture-localhost-8080-1511916281298](https://user-images.githubusercontent.com/26048536/33352024-2e9671d2-d4a7-11e7-8e11-da36aecb0264.png)
#### Questions:
None
